apps/frontend/src/components/global/upload_tabs/LoadFileList.vue

```diff
@@ -83,6 +83,16 @@
           <span>{{ new Date(item.createdAt).toLocaleString() }}</span>
         </template>
         <template #[`item.actions`]="{item}">
+          <v-icon
+            v-clipboard:copy="shareItems(item)"
+            v-clipboard:success="onCopy"
+            v-clipboard:error="onCopyFailure"
+            small
+            class="mr-2"
+            type="button"
+            >mdi-share</v-icon
+          >
+          <button />
```

Comment:
This + the other code added to LoadFileList here should really be in it's own component.